scripts/create-plugin-list.js

```diff
@@ -0,0 +1,10 @@
+import * as corePlugins from '../src/plugins'
+import fs from 'fs'
+import path from 'path'
+
+const corePluginList = Object.keys(corePlugins)
+
+fs.writeFileSync(
+  path.join(__dirname, '..', 'src', 'corePluginList.js'),
```

Comment:
nitpick: the below would work even if we move the file in another location
```suggestion
  path.join(process.cwd(), 'src', 'corePluginList.js'),
```